src/cc_catalog_airflow/dags/provider_api_scripts/wikimedia_commons.py

```diff
@@ -121,7 +121,12 @@ def _get_image_batch(
     )
     image_batch = None
     for _ in range(MEAN_GLOBAL_USAGE_LIMIT):
-        response_json = _get_response_json(query_params, retries=retries)
+        response_json = delayed_requester. \
+            get_response_json(ENDPOINT, retries=retries,
+                              query_params=query_params,
+                              headers=DEFAULT_REQUEST_HEADERS,
+                              timeout=60)
```

Comment:
I suggest the following style:
```suggestion
        response_json = (
            delayed_requester
            .get_response_json(
                ENDPOINT,
                retries=retries,
                query_params=query_params,
                headers=DEFAULT_REQUEST_HEADERS,
                timeout=60
            )
        )
```
In particular, the arguments should be formatted that way to allow a smaller diff later (same reasons as above)